uiowa/uiowa
# Bulk moderation state actions

Resolves #696

Remove non-functional un/publish actions, add functional moderation state transition actions.





# How to test



`git pull && composer install`
Sync a site
Log in and visit /admin/content
Check that the generic "Publish" and "Unpublish" options in the Actions dropdown are no longer present
Select some content and try out the new options--Archive current revision, Unpublish latest revision, Publish latest revision
Masquerade as different roles, and check that you cannot perform operations you don't not have permissions for (eg Editor role cannot Archive content)

Possible followups:
Only show options that are available to the current role? Good UX cleanup, but not default behavior of the Actions menu.
